src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -249,6 +214,12 @@ public ConfigDef config()
     return SnowflakeSinkConnectorConfig.newConfigDef();
   }
 
+  @Override
+  public Config validate(Map<String, String> connectorConfigs) {
+    // TODO cross-fields validation here
+    return super.validate(connectorConfigs);
```

Comment:
I see, so SinkConnector has a validate function right?

Reply:
Yes, correct. We can do cross-fields validation here. I haven't finished this part yet, it is a placeholder.